### ISSUE TITLE
fs_usage reveals Code Helper is working on files outside currently opened projects

### ISSUE THREAD

**Issue by vesper8** [Microsoft/vscode#69277]
I'm on OSX Mojave, using very latest VSCode, my kernel_task was going nuts so I ran this command to see what the hell was going on

```
sudo fs_usage | grep -v 0.0000
```

This showed me that Code Helper was running HFS_update, RdChunkCS and RdMeta[S] on a ton of files that are completely outside my currently opened projects.. There are no symlinks that could be linking them together either.

Let me know what other info I can provide to debug this problem

**Comment by Tyriar** [comment]
Can you elaborate a little on what files outside the projects are being accessed? I can't seem to repro, also would `sudo fs_usage | grep Code` be a better command to check this?

**Comment by vesper8** [comment]
I disabled the intelephense extension which has been causing me so many problems with making Code Helper make my cpu melt down. I also upgraded a bunch of other brew components and ran a SMC and pram reset.. the problem has stopped now thank god.. unfortunately I don't know which of these things have fixed the problem. I'll probably try to re-enable intelephense since it's so useful.. if the problem starts again I'll re-open the issue.
